rfc/5/index.md

```diff
@@ -241,13 +323,12 @@ They:
 - MUST contain the field "type" (string).
 - MUST contain any other fields required by the given "type" (see table below).
 - MUST contain the field "output" (string),
-  unless part of a `sequence` or `inverseOf` (see details).
+  unless part of a wrapper transform (, i.e., [`sequence`](#sequence), [`bijection`](#bijection), [`byDimension`](#bydimension), see details).
 - MUST contain the field "input" (string),
-  unless part of a `sequence` or `inverseOf` (see details).
+  unless part of a wrapper transform (, i.e., [`sequence`](#sequence), [`bijection`](#bijection), [`byDimension`](#bydimension), see details).
 - MAY contain the field "name" (string).
-  Its value MUST be unique across all "name" fields for coordinate transformations.
+  Its value MUST be unique across all "name" fields for all coordinate transformations in the same list.
```

Comment:
```suggestion
  Its value MUST be unique across all "name" fields for all coordinate transformations in the same array.
```
Since the metadata is stored in JSON, I think array is the correct technical term here (https://en.wikipedia.org/wiki/JSON#Data_types).